client/verta/verta/_internal_utils/_utils.py

```diff
@@ -138,7 +138,8 @@ class LazyList(object):
     # number of items to fetch per back end call in __iter__()
     _ITER_PAGE_LIMIT = 100
 
-    _OP_MAP = {'==': _CommonCommonService.OperatorEnum.EQ,
+    _OP_MAP = {'~=': _CommonCommonService.OperatorEnum.CONTAIN,
```

Comment:
"in" is a more common term, if I remember correctly what CONTAINS does

Reply:
I've experienced `CONTAIN` being used more often for fuzzy matching strings (like names).
I could _also_ add `in` as a key for `CONTAIN`.

Reply:
So in python, it's equivalent to `"abc" in project.name`, right? `~=` is very prometheus-based and I'm not sure most people would get it (since the right hand side is different from what prometheus enables, which is full regex)

Reply:
Hmm okay, I see how `~=` would not be the best choice.

I don't see it as _equivalent_ to `"abc" in project.name` since substring checking is different from fuzzy matching (unless that's what the backend does, but I thought it does character substitutions too).

But I'll agree that it's conceptually similar enough to use `in` instead.

Reply:
I believe that's precisely what the backend does.